manifest.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-map
+data:
+  DEBUG: "True"
+  SECRET_KEY: "secret"
+  LOG_LEVEL: "INFO"
+  POSTGRES_DB: "anemperfeina"
+  POSTGRES_USER: "anemperfeina"
+  POSTGRES_PASSWORD: "s3cret"
+  POSTGRES_HOST: "db"
+  POSTGRES_PORT: "5432"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      containers:
+      - name: backend
+        image: geekscat/anem-per-feina:latest
+        command: ["invoke"]
+        args: [
+          "wait-for",
+          "db:5432",
+          "migrate",
+          "runserver",
```

Comment:
For now this is fine, but I will change this for a proper wsgi and we will use it here, doesn't make sense to use the runserver here. But for this initial approach is fine. I'll create a separate issue to do that.

Reply:
I agree, it's the first point in the considerations section:
> current entrypoint is the same than in our composition, in short-term should be moved to a uwsgi server